cmd/trade.go

```diff
@@ -506,7 +506,7 @@ func runTradeCmd(options inputs) {
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
-	userID := "-1" // TODO DS Properly generate and save user ID.
+	userID := "12345" // TODO DS Properly generate and save user ID.
```

Comment:
this seems like debug code since we still want the userID to be -1 for now (disabled), can you remove so it doesn't go into `master` (affects local dev builds)?

Reply:
Thanks for catching!